examples/multitask-monorepo/main.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_key_pair" "root" {
```

Comment:
Name of the file doesn't look right. Needs to be `examples/multistack-monorepo/main.tf`